recipes/sphinxcontrib-getthecode/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "sphinxcontrib-getthecode" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/FabriceSalvaire/sphinx-getthecode/archive/v{{ version }}.tar.gz
+  # run openssl sha256 <file name>
```

Drop example comments

```suggestion
  # run openssl sha256 <file name>
```

---

just a comment